src/components/PrescriptionsComponent.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+
+const Prescriptions = () => {
+    const [prescriptions, setPrescriptions] = useState<Prescription[]>([]);
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState(null);
+
+    // Retrieve the userId from local storage
+    const userId = localStorage.getItem("userId"); // Adjust as necessary based on how you're storing userId
+
+    // Retrieve the cached medicines from localStorage
+    const cachedMedicines: MedicineDbModel[] = JSON.parse(
+        localStorage.getItem("medicines") || "[]"
+    );
+
+    // Helper function to get the medicine name by ID
+    const getMedicineNameById = (id: string): string => {
+        const medicine = cachedMedicines.find((med) => med._id === id);
+        return medicine ? medicine.name : "Unknown Medicine";
+    };
+
+    useEffect(() => {
+        const fetchPrescriptions = async () => {
+            try {
+                // Ensure you have the token available from your authentication process
+                const token = localStorage.getItem("authToken"); // Adjust where you get the token from
+
+                const response = await axios.get(
+                    `https://sea-turtle-app-9l4ak.ondigitalocean.app/api/users/${userId}/getPrescriptions`,
```

Comment:
If you could I would suggest swapping out the base url to an env variable rather than hard coding it